hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.less

```diff
@@ -79,4 +79,16 @@
 
 .ant-drawer-open .ant-drawer-mask {
   opacity: 0 !important;
+}
+
+div.svg-container {
+  width: 1400px !important;
+}
+
+svg.main-svg {
+  width: 1400px !important;
+}
+
+g.legend {
+  transform: translate(650px, 100px) !important;
```

Comment:
Same comment as above.

Reply:
@devmadhuu 
We have tested different screensize and Responsive web also. Attached new screenshots with testing result.